app/scripts/controllers/transactions/index.js

```diff
@@ -2074,8 +2131,28 @@ export default class TransactionController extends EventEmitter {
       TRANSACTION_TYPES.SWAP_APPROVAL,
     ].includes(type);
 
-    let transactionType = TRANSACTION_TYPES.SIMPLE_SEND;
+    const contractMethodNames = {
+      CONTRACT_INTERACTION: 'Mint Collectibles',
+      APPROVE: 'Approve',
```

Comment:
Is the `APPROVE` property the only property from this object that is being used?

Reply:
Yes @danjm, the `APPROVE` property is the only property from this object that is being used. What is your suggestion here, can I remove all other methods and just to stay the `APPROVE` property because just this property is being used in the code?

Reply:
@danjm Done [here](https://github.com/MetaMask/metamask-extension/pull/15289/commits/00676999f69009ad1d132e9da0af0664cfaecc70).